### ISSUE TITLE
[A11y 8/25] Questions about functionality

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/iron-list#83]
These are more of a question of how this should function. 

With VoiceOver/Chrome: When navigating through the top portion of this, I just hear ""selected button"" for the go back button and the search button and menu button.  I also cannot get to the longer paragraphs below the large  blue top header. Should I be able to? 

ChromeVox / Chrome OS - not able to access the stars on the right side. Should I be able to? They have no labels. If not, feel free to close this out.

**Comment by blasten** [comment]
Users can now navigate through the content and select/deselect items in the example (demos/selection.html)
